<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-GK61F68" name="runnerIO.Runner" tests="1" failures="1" timestamp="2020-12-21T19:19:07 IST" time="49.904" errors="0">
  <testcase name="runScenario" time="49.904" classname="runnerIO.Runner">
    <failure type="java.lang.AssertionError" message="expected [Testyantra] but found [Testyantra450]">
      <![CDATA[java.lang.AssertionError: expected [Testyantra] but found [Testyantra450]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at stepDefinitions.OrganizationStepDef.i_navigate_to_organization_information_and_verify(OrganizationStepDef.java:49)
at âœ½.I navigate to organization information and verify(file:///C:/Users/Nitheesha/git/Sdet8Bdd/SDET8BDD/./src/test/resources/Features/Organization.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- runnerIO.Runner -->
